### PR TITLE
Icon Updates

### PR DESCRIPTION
**Pull request by rossmoody**
Fix https://github.com/brave/brave-browser/issues/1434
Fixes https://github.com/brave/brave-browser/issues/1390
Fixes https://github.com/brave/brave-browser/issues/1203
Fixes https://github.com/brave/brave-browser/issues/1385

- general icon aesthetic and ver fine polish updates
- adjusted menu to kabob instead of dots
- settings omnibox lion now is full color
- house is more housey
- omnibox https security chip icons are revised to spec
- adjusted autoplay icon from extension to a play button icon
- incognito icon went from weird robber guy to cool sunglasses
- Page action icons (magnify, find-in-page, autoplay, cookies, generic extension)
- Some default favicon but not the real tab bar default favicon
- removed the unused VR icon

Use new icon for blocked autoplay graphic, since the usage of it is a new feature.

image
image
image

### CODE REVIEW

**Comment by simonhong** [comment]
Is this still WIP?

**Comment by rossmoody** [comment]
@simonhong this one is ready for a review. had a few additional permissions icons i wanted to squeeze in

File: browser/ui/content_settings/brave_autoplay_blocked_image_model.cc [hunk 1]
```diff
@@ -32,7 +32,7 @@ void BraveAutoplayBlockedImageModel::UpdateFromWebContents(
 
   set_visible(true);
   const gfx::VectorIcon* badge_id = &kBlockedBadgeIcon;
-  const gfx::VectorIcon* icon = &kExtensionIcon;
+  const gfx::VectorIcon* icon = &kPlayArrowIcon;
```
Comment (petemill) [inline browser/ui/content_settings/brave_autoplay_blocked_image_model.cc]: I think I'll change this to add a new icon

File: vector_icons/chrome/app/vector_icons/default_favicon.icon [hunk 2]
```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (petemill) [inline vector_icons/chrome/app/vector_icons/default_favicon.icon]: This icon doesn't seem to be working @rossmoody - where did you intend it to go - on the tab?
image

Comment (petemill) [reply vector_icons/chrome/app/vector_icons/default_favicon.icon]: So looks like this icon is only included on the macOS build, possibly related to touchbar, though I couldn't get it to show up. Let's keep it in, but if we also want to change the default favicon, I think we'll need to provide PNGs @rossmoody . I'll create a separate issue

Comment (petemill) [reply vector_icons/chrome/app/vector_icons/default_favicon.icon]: https://github.com/brave/brave-browser/issues/1433

**Comment by petemill** [comment]
Passed internal security review

**Comment by petemill** [comment]
0.56.x 5cb7a906
0.55.x d278e117

**Pull request merged**